ONSdigital/response-operations-ui
# Remove change of username, update username on email change

# What and why?
This PR removes the ability for user to change their username, instead merging the functionality in with email change (as the 2 field are now tied together)

# How to test?
From the UI login as normal then go to My Account, make sure you can change your email but not username and the order matches the card. Select email change and then change it to something new, make sure the verification email is sent and when you click on it both the user_name and email are updated. Now login with your new username/email to make sure it works. Once confirmed try some unhappy paths, i.e change your email to one that already exists, hit the old end point for changing username etc

N.B I have reduced logging for successful paths, I only really want to know on error
# Trello
